webapp/components/wpt-insights.html

```diff
@@ -0,0 +1,68 @@
+<!--
+Copyright 2018 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="../bower_components/paper-card/paper-card.html">
+<link rel="import" href="info-banner.html">
+<link rel="import" href="product-info.html">
+<link rel="import" href="wpt-flags.html">
+
+<dom-module id="wpt-insights">
+  <template>
+    <style>
+      info-banner {
+        margin: 0;
+      }
+      paper-card {
+        margin: 1em;
+      }
+    </style>
+    <template is="dom-repeat" items="[[insights]]" as="item">
+      <paper-card>
+        <div class="card-content">
+          <h3>[[item.name]]</h3>
+          <info-banner>
+            <a class="query" href="[[item.url]]">
+              [[item.query]]
+            </a>
+          </info-banner>
+          <p>
+            [[item.desc]]
+          </p>
+        </div>
+      </paper-card>
+    </template>
+  </template>
+  <script>
+    /* global ProductInfo, WPTFlags */
+    class Insights extends ProductInfo(WPTFlags(window.Polymer.Element)) {
+      static get is() {
+        return 'wpt-insights';
+      }
+
+      get insights() {
+        const browsers = ['chrome', 'edge', 'firefox', 'safari'];
+        return browsers.map(b => {
+          const others = browsers.filter(o => o !== b);
+          const othersFailing = others
+            .map(o => `(${o}:pass|${o}:ok)`)
+            .join(' ');
+          const query = `!${b}:pass !${b}:ok ${othersFailing}`;
+          const url = new URL('/results/', window.location);
+          url.searchParams.set('q', query);
+          return {
+            name: `${this.displayName(b)}-only failures`,
+            url,
+            query,
+            desc: `Tests that are failing in ${this.displayName(b)}, but passing in the other browsers (${others.map(this.displayName).join(', ')})`,
```

Comment:
nit: Should `this.displayName` be bound before being passed to `map()`?

Reply:
Nah, it doesn't use `this`, it's just looking up a key in a map.